app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Posts</h1>
+<%= render partial: "users/username", locals: { user: @posts[0].author} %>
+<ul class='user-posts'>
+<% @posts.each_with_index do |post, index| %>
+  <li>
+  <h4>Post #<%=index + 1%></h4>
+  <p><%=post.text%></p>
+  <span>Comments: <%= post.comments_counter%>, Likes: <%= post.likes_counter%></span>
+  </li>
+  <%= render partial: "comment", locals: { post: post } %>
```

Comment:
- [ ] Please use the custom method `five_recent_comments` here that you created to display only five comments instead of displaying all as requested in the project requirements.